src/components/Gesture.svelte

```diff
@@ -271,37 +271,36 @@
 </script>
 
 <!-- Recording countdown popup -->
-<StandardDialog isOpen={showCountdown || isThisRecording} onClose={cancelRecording}>
-  <div class="flex flex-col items-center gap-8 w-120">
-    <h2 class="text-xl font-bold self-start">
-      {$t('content.data.recordingDialog.title', { values: { action: $nameBind } })}
-    </h2>
-    <div class="flex flex-col space-y-3 self-center items-center justify-center">
-      <div class="flex justify-center">
-        <p class="text-lg px-10 text-center">
-          {$t('content.data.recording.description')}
+<StandardDialog
+  isOpen={showCountdown || isThisRecording}
+  onClose={cancelRecording}
+  class="flex flex-col items-center gap-8 w-120"
+  titleClass="text-xl font-bold self-start"
+  title={$t('content.data.recordingDialog.title', { values: { action: $nameBind } })}>
```

Comment:
I used`title` instead of `titleId` as prop for the melt dialog due to this usage where the translation string might need values. I also added a`titleClass` prop for melt dialog because of this use case where the heading styling is different

Reply:
Props no longer required.